.travis.yml

```diff
@@ -1,15 +1,14 @@
 language: java
 jdk:
-  - openjdk7
+  - openjdk8
 os:
   - linux
-dist: trusty
 cache:
   directories:
     - ~/.m2/repository
 install:
-  - mvn install -DskipTests=true -Dmaven.javadoc.skip=true --quiet --batch-mode --show-version --activate-profiles travis
+  - mvn install -DskipTests=true -Dmaven.javadoc.skip=true --quiet --batch-mode --show-version
 before_script:
   - echo "MAVEN_OPTS=${TEST_MEM_SETTINGS}" > ~/.mavenrc
 script:
-  - mvn cobertura:cobertura coveralls:report --batch-mode --activate-profiles travis -DrepoToken=${COVERALLS_REPO_TOKEN} -Dtest.arguments="${TEST_MEM_SETTINGS}"
+  - mvn package jacoco:report coveralls:report --batch-mode -Dtest.arguments="${TEST_MEM_SETTINGS}" -P coveralls --quiet
```

Comment:
Please use the long name for the arguments as they are more readable (i.e. `--activate-profiles` instead of `-P`). 

The coveralls repo token argument needs to be put back.

Why did you add `--quiet`?

Reply:
good catch, didn't realize I removed the token.
Added quiet because travis was failing with : `The job exceeded the maximum log length, and has been terminated.`, it doesn't look like it helped much. Will go ahead and revert this

Reply:
ah right - I had copied that directly from the other PR. Do we need the token call still?